.github/workflows/create-release.yaml

```diff
@@ -0,0 +1,103 @@
+name: Release Collection
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  create_collection_release:
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release')
+    runs-on: ubuntu-latest
+    environment: fusion-env
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get version from galaxy.yml
+        run: |
+          RELASE_VERSION=$(grep 'version:' galaxy.yml | awk '{print $2}')
+          echo "RELASE_VERSION=$RELASE_VERSION" >> $GITHUB_ENV
+
+      - name: Check if tag exists
+        env:
+          GITHUB_TOKEN: ${{ github.TOKEN }}
+        run: |
+          trap "exit 0" EXIT
+          response=$(gh api -H "Accept: application/vnd.github+json" /repos/${{ github.repository }}/git/ref/tags/$RELASE_VERSION)
+          if [[ "$response" == *"$RELASE_VERSION"* ]]; then
+            trap "exit 1" EXIT
+            echo "Error: Tag $RELASE_VERSION already exists"
+            exit 1 
+          fi
+
+      - name: Extract changelog
+        run: |
+          awk -v version="$RELASE_VERSION" '
+          BEGIN { targetVersionFound = 0; previousVersionFound = 0 }
+          {
+            if (match($0, "^v"version) > 0) {
+              targetVersionFound = 1;
+            } else if (targetVersionFound && match($0, "^v") > 0) {
+              previousVersionFound = 1;
+            }
+
+            if (targetVersionFound && !previousVersionFound) {
+              print $0;
+            }
+          }
+          ' CHANGELOG.rst > changelog
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ansible ansible-lint yamllint ansible-galaxy ansible-galaxy-importer
+
+      - name: Build Ansible Collection
+        run: ansible-galaxy collection build
+
+      - name: Create a release
+        env:
+          GITHUB_TOKEN: ${{ github.TOKEN }}
+        run: gh release create "${{ env.RELEASE_VERSION }}" --title "Fusion Collection v${{ env.RELEASE_VERSION }}" --notes-file changelog
```

Comment:
So we only need the changelog for this release version., not the whole changelog

Reply:
I have a step for extracting change log 
https://github.com/DenDen039/Fusion-Collection/blob/0f570add89294731045d7e165ca9a091477c7ff5/.github/workflows/create-release.yaml#L34

Reply:
Cool